packages/calcite-components/src/components/stepper/stepper.tsx

```diff
@@ -112,17 +113,19 @@ export class Stepper {
 
   render(): VNode {
     return (
-      <slot
-        onSlotchange={(event: Event) => {
-          const items = (event.currentTarget as HTMLSlotElement)
-            .assignedElements()
-            .filter((el) => el?.tagName === "CALCITE-STEPPER-ITEM");
-          const spacing = Array(items.length).fill("1fr").join(" ");
-          this.el.style.gridTemplateAreas = spacing;
-          this.el.style.gridTemplateColumns = spacing;
-          this.setStepperItemNumberingSystem();
-        }}
-      />
+      <Host aria-label={"Progress steps"} role="region">
+        <slot
+          onSlotchange={(event: Event) => {
+            const items = (event.currentTarget as HTMLSlotElement)
```

Comment:
We have a dom util helper for getting the assigned elements.

Reply:
Good catch, updated this.